docs/search_syntax.md

```diff
@@ -0,0 +1,294 @@
+For the ```numeric``` attributes you can use the following comparison operator:
+- ``` == ```
+- ``` != ```
+- ``` > ```
+- ``` >= ```
+- ``` < ```
+- ``` <= ```
+
+### Boolean operations
+For the ```boolean``` attributes you can use the following comparison operator:
+- ``` == ```
+- ``` != ```
+
+#### Implicit Boolean comparison
+You can also use boolean attributes directly in a conditional statement without any comparison operator. 
+
+Equivalent to comparing the attribute to True.
+
+```python
+run.archived  # Checks if run.archived is equal to True
+```
+Equivalent to comparing the attribute to False.
+```python
+not run.archived  # Checks if run.archived is equal to False
+```
+### Logical operations
+You can create complex search queries combining multiple conditions with logical operators.
+- ``` and ```
+- ``` or ```
+- ``` not ```
+
+
+
+## Search run examples
+
+### Example with ```run.name``` (string)
+Select only the runs where the name exactly matches "TestRun1"
+
+```python
+run.name == "TestRun1"
+```
+
+Select only the runs where the name is different from "TestRun1"
+```python
+run.name != "TestRun1"
+```
+
+Select only the runs where "Run1" is contained in the run name
+```python
+"Run1" in run.name
+```
+
+Select only the runs where the name starts with "Test"
+```python
+run.name.startswith('Test')
+```
+
+Select only the runs where the name ends with "Run1"
+```python
+run.name.endswith('Run1')
+```
+
+### Example with ```run.duration``` (numeric)
+
+Select only the runs where the duration is exactly 60
+
+```python
+run.duration == 60
+```
+
+
+Select only the runs where the duration is not 60
+```python
+run.duration != 60
+```
+
+Select only the runs where the duration is greater than 60
+```python
+run.duration > 60
+```
+
+Select only the runs where the duration is greater or equal to 60
+```python
+run.duration >= 60
+```
+
+Select only the runs where the duration is less than 60
+```python
+run.duration < 60
+```
+
+Select only the runs where the duration is less or equal to 60
+```python
+run.duration <= 60
+```
+
+### Example with ```run.archived``` (boolean)
+Select only the runs where the archived attribute is true
+```python
+run.archived == True
+```
+
+Select only the runs where the archived attribute is not true
+```python
+run.archived != True
+```
+
+### Run parameters
+Run parameters can be accessed via parameters.
+![FastTrackML Run List, param filter](images/search_runs_param_filter.png)
+
+### Filtering Runs with Unset Parameters
+
+To filter runs based on whether an attribute is not set, you can use the following syntax:
+
+```python
+run.attribute is None
+```
+This expression will return runs for which the specified attribute is not defined.
+
+Showing all the runs
+![FastTrackML Run List](images/search_runs.png)
+
+Showing only the runs where param1 is not set
+![FastTrackML Run List of not set param](images/search_runs_none_param.png)
+
+### Filter Runs using Regular Expressions
```

Comment:
regex is not specific to runs, it works for any string (e.g. `metric.name`) — it may unclear to users that it can work anywhere, maybe we should add regex ops to the string ops at the top and add an example of filtering metrics using regex?

Reply:
Actually regex are not working at the moment with metric search